docs/manifest.json

```
@@ -16,15 +16,16 @@
       ["Remix", "/quickstarts/remix"],
       ["Gatsby", "/quickstarts/gatsby"],
       ["RedwoodJS", "/quickstarts/redwood"],
+      ["Expo", "/quickstarts/expo"],
       "# Frontend",
       ["JavaScript", "/quickstarts/javascript"],
       ["React", "/quickstarts/react"],
       "# Backend",
       ["Express", "/quickstarts/express"],
       ["Fastify", "/quickstarts/fastify"],
       "# Example Apps",
-      ["Next.js starter repo", "/quickstarts/external-link"],
-      ["Remix starter repo", "/quickstarts/external-link"]
+      ["Next.js starter repo", "https://github.com/clerkinc/clerk-nextjs-starter"],
```

This is the newer one:
```suggestion
      ["Next.js starter repo", "https://github.com/clerkinc/clerk-next-app"],
```